mldsa/fips202/fips202x4.c

```diff
@@ -7,6 +7,7 @@
 #include "../common.h"
 
 #include <string.h>
+#include "../sign.h"
```

Comment:
nit: sign.h should not be included here. common should be sufficient and it's already included.